libs/libencrypt/config.h.in

```diff
@@ -0,0 +1,9 @@
+ 
```

Comment:
I suggest to move the config.h.in, config.txt, main.cpp to a subdirectory ``test``. Following the convention as other vtr library: 

https://github.com/verilog-to-routing/vtr-verilog-to-routing/tree/ec634df07a5be186a0e7b66f2789c4daf2e4b44c/libs/libvtrutil